.azure-pipelines/pipelines.yml

```diff
@@ -23,6 +23,28 @@ stages:
     - template: bazel.yml
       parameters:
         ciTarget: $(CI_TARGET)
+- stage: check_salvo_remote
+  dependsOn: []
+  pool:
+    vmImage: "ubuntu-20.04"
```

Comment:
How about to update image to `ubuntu-22.04`? It's a latest long-term support system, which Envoy is based on.

Reply:
Good point, done in 98c5a7bfbceea90393c0e43be389c5269d7d7a65.